applications/nrf5340_audio/src/bluetooth/bt_management/scanning/bt_mgmt_scan_for_conn.c

```diff
@@ -195,6 +261,43 @@ static void scan_recv_cb(const struct bt_le_scan_recv_info *info, struct net_buf
 	}
 }
 
+static void conn_in_set_check(struct bt_conn *conn, void *data)
+{
+	int ret;
```

Comment:
Should do some NULL checks here.

Reply:
We could check that data is not NULL, but conn we do not need to check since this is provided by host through it's conn list.